library/std/src/time/tests.rs

```diff
@@ -12,6 +12,15 @@ macro_rules! assert_almost_eq {
     }};
 }
 
+#[test]
+#[cfg(all(target_arch = "aarch64", target_os = "macos"))]
```

Comment:
```suggestion
```
There is nothing unique to macOS or even aarch64 about this test.
It should pass on every platform.

Reply:
I agree, I changed it. Thanks